frontend/src/metabase/visualizations/components/ChartSettingsWidgetPopover.styled.tsx

```diff
@@ -1,10 +1,36 @@
 import styled from "@emotion/styled";
+import Radio from "metabase/core/components/Radio";
 
-export const PopoverRoot = styled.div`
-  padding: 1.5rem 0;
+import { color } from "metabase/lib/colors";
+
+interface PopoverRootProps {
+  noTopPadding: boolean;
+}
+
+export const PopoverRoot = styled.div<PopoverRootProps>`
+  padding: 1.5rem 0 1rem;
   overflow-y: auto;
   max-height: 600px;
   min-width: 336px;
+
+  ${({ noTopPadding }) => noTopPadding && "padding-top: 0;"}
+`;
+
+export const PopoverTabs = styled(Radio)`
+  padding: 0;
+  border-bottom: 1px solid ${color("border")};
+  margin: 0 1rem 1rem 1rem;
+
+  ${Radio.RadioLabelVariants.join(", ")} {
```

Comment:
I swear I've never seen anything like this! It's cool knowing that emotion components when turning into a string would be their runtime class names.

<img src="https://media3.giphy.com/media/26ufdipQqU2lhNA4g/giphy.gif?cid=ecf05e47oaebs1iztt75o54r0c7srecmjj5ase16axhra3tc&rid=giphy.gif&ct=g" />